tyk-pro/templates/secret-dashboard.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-login-details
+  labels:
+    app: login-details-{{ include "tyk-pro.fullname" . }}
+    chart: {{ include "tyk-pro.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  TYK_PASS: {{ randAlphaNum 10 | b64enc | quote }}
```

Comment:
I think it would be useful for the user to set this optionally through a value too, while defaulting to a random.